src/main/java/bio/terra/service/snapshotbuilder/SelectAlias.java

```diff
@@ -0,0 +1,22 @@
+package bio.terra.service.snapshotbuilder;
+
+import bio.terra.common.CloudPlatformWrapper;
+import bio.terra.service.snapshotbuilder.query.FieldVariable;
+import bio.terra.service.snapshotbuilder.query.SelectExpression;
+
+/** Wrap a field variable with an alias for use in a SELECT. */
```

Comment:
Does this not already exist? In FieldVariable you can add an alias

Reply:
That's true, but the alias support in `FieldVariable` doesn't work correctly for this case. It wasn't clear to me if the alias support was broken completely or if it just didn't work here. I didn't want to hold up this work on it, but I agree we need to either fix `alias` or get rid of it. I'll keep looking at it now but if I can't fix it I'll create a ticket to go back and clean it up.